README.md

```diff
@@ -3,289 +3,158 @@
-
-* Application projects
-
-  Application projects are self-contained and intended to be deployed as a standalone website. Application projects contain elements in a `src/` folder and import their dependencies with absolute paths, or relative paths that reference folders inside the project folder.
-
-### Application Styles
-
-Polymer-CLI currently supports two styles of applications:
-
-  * Monolithic applications, which have a single entrypoint (usually index.html) and eagerly import all dependencies.
-
-  * "App shell" applications, which have a very lightweight entrypoint, an app-shell with startup and routing logic, and possibly lazy loaded fragments.
-
-### App-shell Structure
-
-App-shell apps are currently the preferred style for Polymer CLI, and most commands are being optimized to support them. App-shell apps usually have client-side routing (see the [app-route](https://github.com/PolymerElements/app-route) element), and lazy load parts of the UI on demand.
-
-Polymer-CLI supports this style by understand these different types of files:
+```bash
+$ yarn add global polymer-cli
+# or...
+$ npm install -g polymer-cli
+```
 
-  * entrypoint - The first file served by the web server for every valid route (usually index.html). This file should be very small, since it may not cache well and must reference resources with absolute URLs, due to being served from many URLs.
-  * shell - The actual app shell, which includes the top-level logic, routing, and so on.
-  * fragments - lazy loaded parts of the application, typically views and other elements loaded on-demand.
+For best results and a faster installation, we recommend installing with [yarn](https://yarnpkg.com/en/).
 
 ## Configuration
 
-The project files are specified either as global flags: `--entrypoint`, `--shell` and zero or more `--fragment` flags, or in a `polymer.json` configuration file.
+An application can be configured for all commands via global flags: `--entrypoint`, `--shell`, etc.
 
-### polymer.json
+We recommend saving your configuration to a `polymer.json` configuration file in your project so that they can be read automatically for every command. Other project settings, like build and lint rules, can also be defined here. Read the [polymer.json spec](https://www.polymer-project.org/2.0/docs/tools/polymer-json) for a full list of all supported fields.
 
-You can specify the project files in `polymer.json` so that commands like `polymer build` work without flags:
+Below is an example of a simple `polymer.json` application configuration:
 
 ```json
 {
   "entrypoint": "index.html",
-  "shell": "src/my-app/my-app.html",
+  "shell": "src/my-app.html",
   "fragments": [
-    "src/app-home/app-home.html",
-    "src/app-view-1/app-view-1.html",
+    "src/app-home.html",
+    "src/app-view-1.html",
+    "src/app-view-2.html",
+    "src/app-view-3.html"
   ],
   "sources": [
     "src/**/*",
     "images/**/*",
     "bower.json"
```

Comment:
Not until we remove support from polyserve :(
https://github.com/Polymer/polyserve/blob/84752470189f75efa11f2a2dd75572a8b84e292c/src/test/make_app_test.ts#L68